addons/gdUnit4/src/update/GdUnitUpdate.gd

```diff
@@ -189,37 +189,13 @@ func temp_dir() -> String:
 
 func create_temp_dir(folder_name :String) -> String:
 	var new_folder := temp_dir() + "/" + folder_name
-	delete_directory(new_folder)
+	GdUnitFileAccess.delete_directory(new_folder)
```

Comment:
Good catch to use the existing `GdUnitFileAccess.delete_directory` and apply the fix in this implementation 👍